app/assets/javascripts/bulk_updates.coffee

```diff
@@ -345,6 +349,13 @@ $(document).ready ->
         else
           error.insertAfter(element)
 
+
+    # on load, disable the inputs from partials that are hidden
+    $('.bulk-update-partial').each ->
+      if $(this).hasClass('is-hidden')
+        $update_field_selector = "#" + $(this)[0].id
+        hideAndClear($update_field_selector)
```

Comment:
I think I would shrink this down to one line, `hideAndClear("##{$(this)[0].id}")` (maybe that works?)

Reply:
Yeah, I think that'll work. I'll make the change and keep it in if it works.